CRM/Core/BAO/UserJob.php

```diff
@@ -154,10 +154,9 @@ private static function findUserJobId(string $queueName): ?int {
    */
   public function addSelectWhereClause(string $entityName = NULL, int $userId = NULL, array $conditions = []): array {
     $clauses = [];
-    if (!\CRM_Core_Permission::check('administer queues')) {
-      // @todo - the is_template should really be prefixed. We need to add support
-      // for that in the compiler & then this would be `{table}.is_template`
-      $clauses['created_id'] = '= ' . (int) CRM_Core_Session::getLoggedInContactID() . ' OR is_template = 1';
+    if (!\CRM_Core_Permission::check('administer queues', $userId)) {
```

Comment:
@colemanw actually just thinking that the prefix thing should not have come out of the comment but also - should it be
```
`$userID = (int) ($userId ?: CRM_Core_Session::getLoggedInContactID()); 
$clauses['created_id'][] = '= ' . $userID . ' OR {is_template} = 1';
```

In practice the user ID is the logged in contact ID - but the point of passing it is presumable cos it might not be?